eq-author/src/App/section/Design/SectionEditor/SectionEditor.test.js

```diff
@@ -136,6 +146,28 @@ describe("SectionEditor", () => {
     ).toBe(false);
   });
 
+  it("should show an error when there is a validation error", () => {
+    const getValidationError = jest.fn().mockReturnValue("Validation error");
+    const wrapper = render({
+      section: {
+        ...section1,
+        title: "",
+      },
+      getValidationError,
+    });
+    expect(
+      wrapper
+        .find("[testSelector='txt-section-title']")
+        .prop("errorValidationMsg")
+    ).toBe("Validation error");
+
+    expect(getValidationError).toHaveBeenCalledWith({
+      field: "title",
+      message:
+        "Enter a section title. If section navigation is not required, disable in 'settings'.",
```

Comment:
Seems weird to be calling `getValidationError` passing in the message, only to return the message, but I appreciate why it's being done.